src/views/Explorer/FilesView/FileComponent.js

```diff
@@ -70,6 +111,8 @@ const fileComponentSource = {
     endDrag(props, monitor, component) {
         // console.log("EndDrag", monitor.getDropResult());
         console.log(props, "Component:", component);
+        ReactDOM.render(modalElement, document.getElementById("modalElement"));
```

Comment:
The problem is that here we'll have to wait for the component to be rendered and processed completely and only after that getting the value of `operation` will be possible.
As of now, before the element is rendered completely, the code runs to `console.log(operation);` and then to try block ignoring the `modalElement`.......can Async-Await deal with this problem?

Reply:
One way to go about it could be to delay the calling of `performCopyMoveOperation()` till the modal opens and closes.